src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -303,9 +304,62 @@ public void updateReplicateStatus(String chainTaskId,
         }
 
         if (!canUpdate) {
+            log.warn("Cannot update replicate {}",
+                    getStatusUpdateLogs(chainTaskId, replicate, statusUpdate));
+            return Optional.of(ReplicateStatusUpdateError.GENERIC_CANT_UPDATE);
+        }
+
+        return Optional.empty();
```

Comment:
I would change method name (here canUpdate returns a cantStatus) or/and add a "sucess" status for the canUpdate method (instead of empty)

Reply:
I prefer the second solution. A `NO_ERROR` stratus seems great.